financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsRepository.kt

```diff
@@ -35,3 +39,76 @@ internal interface FinancialConnectionsRepository {
         sessionId: String
     ): MixedOAuthParams
 }
+
+internal class FinancialConnectionsRepositoryImpl @Inject constructor(
```

Comment:
Merged interface and implementation on the same file, as we do with the rest of repos.